src/test/java/com/wikia/webdriver/common/contentpatterns/MercuryWikis.java

```diff
@@ -12,4 +12,5 @@ private MercuryWikis() {}
   public static final String MERCURY_NTV_CC = "mercuryntvcc";
   public static final String MERCURY_NTAV_CC = "mercuryntavcc";
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Move this variable to comply with Java Code Conventions. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1213)